addrmgr/addrmanager.go

```diff
@@ -670,7 +670,10 @@ func (a *AddrManager) AddressCache() []*wire.NetAddress {
 	allAddr := make([]*wire.NetAddress, 0, addrIndexLen)
 	// Iteration order is undefined here, but we randomise it anyway.
 	for _, v := range a.addrIndex {
-		allAddr = append(allAddr, v.na)
+		// Skip low quality addresses.
+		if !v.isBad() {
```

Comment:
i'd prefer   ```if v.isBad() { continue }```

Reply:
OK, changed.

Reply:
I see it's already been changed, but just wanted to comment that I agree.  We prefer to keep the primary logic as "unindented" as possible in general.

Reply:
Makes sense, thanks for the explanation! "keeping the primary logic as unindented as possible " is a good heuristic.

I realized that it correlates nicely with the standard Go idiom for error handling:

```
err := foo()
if err != nil {
    return err
}
// Real logic lives here
```

Would it make sense to add your remark to the Code Contribution Guidelines?